fetch.bs

```diff
@@ -1892,6 +1892,24 @@ source of security bugs. Please seek security review for features that deal with
  <a for="URL serializer"><i>exclude fragment</i></a> set to true.
 </ol>
 
+<p>To check <dfn export>Cross-Origin-Embedder-Policy allows credentials</dfn>, given a <a
+for=/>request</a> <var>request</var>, run theses steps:
```

Comment:
Let's avoid having a new line in an inline element.

Reply:
Note that this rule differs between whatwg/html and whatwg/fetch :(. We are sorry about that.

Reply:
Done.